src/content/docs/implementation/setup/content-delivery-network.mdx

```diff
@@ -550,7 +550,7 @@ HTTP/2 200
 Validate that all requests are using `HTTP/2` or `HTTP/3` connections.
 
 ```bash
-curl -Is -L https://aemshop.net | grep location
+curl -Is -L https://www.aemshop.net | grep location
```

Comment:
Nope, this one should be aemshop.net, because the response isa redirect, thus the `location` header :) 
```suggestion
curl -Is -L https://aemshop.net | grep location
```